src/modules/tenant/service.js

```diff
@@ -40,6 +40,11 @@ export default class TenantService extends Service {
 
 		// In case if the tenant list from userinfo is undefined or empty remove tenant parameter from URL
 		if ((!tenants_list) || (tenants_list.length == 0)) {
+			this.App.Store.dispatch({
+				type: types.TENANTS_CHANGED,
+				undefined,
+				current: null
+			});
```

Comment:
Is this needed?

Reply:
@ateska Yes, is an important part of this change

Reply:
@ateska we need it at a time when no tenants are assigned. And we write it as tenant - undefined to distinguish situations when to show this Logout card

Reply:
I still don't get it.
Let's discuss this latest on review, pls.